lib/main-menu.js

```diff
@@ -153,6 +153,19 @@ var view = {
   label: 'View',
   submenu: [
     {
+      label: 'Toggle Full Screen',
+      accelerator: (() => {
+        if (process.platform === 'darwin') return 'Ctrl+Command+F'
+        return 'F11'
```

Comment:
Could you change like below?

```
accelerator: OSX ? 'Control + Command + F' : 'F11',
```

And I'm not sure F11 is available on Linux or not. Do you have any ideas?